.devcontainer/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye@sha256:105bf6a63ab025206f019a371a735fec6553db0be520030c7a2fd0e002947232
+
```

Comment:
In an earlier version of this patch, this also had some Python env vars set up and an `EXPOSE`, similar to the main `docker/Dockerfile`.
```
ENV PYTHONUNBUFFERED=1 \
    PYTHONDONTWRITEBYTECODE=1 \
    PYTHONPATH=/app/ \
    PORT=8000

EXPOSE $PORT
```

Since we're basing this container off the `test` container based on `.devcontainer/docker-compose.yml`, is this no longer necessary?

I can confirm that when I open a shell in the dev container, that these env vars are indeed set.

Reply:
I don't see any of those 4 env vars set in my dev container, and i don't see any exposed ports for it either

Reply:
I originally simply copied [the environment variables from the Tecken Dockerfile](https://github.com/mozilla-services/tecken/blob/ddde451cff69026e8f6c479e6242dc051c6578d8/docker/Dockerfile#L17-L20) into this Dockerfile, but when Bianca asked me why I did that I realized there is no good reason, so I removed them again.